sdcflows/interfaces/fmap.py

```diff
@@ -23,6 +24,111 @@
 
 LOGGER = logging.getLogger('nipype.interface')
 
+class ProcessPhasesInputSpec(BaseInterfaceInputSpec):
```

Comment:
```suggestion
class _ProcessPhasesInputSpec(BaseInterfaceInputSpec):
```

I've started to mark the Input/OutputSpecs as private so that they have reduced visibility. We will see this more often I believe.